specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/examples/createWorkspace.json

```diff
@@ -88,7 +88,11 @@
                 "status": "Approved"
               }
             }
-          ]
+          ],
+          "notebookInfo": {
```

Comment:
Why are we introducing this field to this example? The model validation is reporting that there is not such field in workspace.
Could you double check about this?
Similar questions in all the examples above.

Reply:
Thanks for the catch! I corrected the ResourceId fields in all examples. It's not a ARM ResourceId actually, it's a dataplane resourceId that used to initialize the notebook